scipy/stats/tests/test_multivariate.py

```diff
@@ -216,7 +216,7 @@ def test_broadcasting(self):
                 # Repeat for cdf
                 actual = multivariate_normal.cdf(X[i, j], mean, cov)
                 desired = multivariate_normal.cdf(X, mean, cov)[i, j]
```

Comment:
I'm not sure if such nice x exists --- the computation appears to use a randomized algorithm.